modules/core/02-client/keeper/keeper.go

```diff
@@ -389,6 +388,6 @@ func (k Keeper) GetAllClients(ctx sdk.Context) []exported.ClientState {
 // ClientStore returns isolated prefix store for each client so they can read/write in separate
 // namespace without being able to read/write other client's data
 func (k Keeper) ClientStore(ctx sdk.Context, clientID string) sdk.KVStore {
-	clientPrefix := []byte(fmt.Sprintf("%s/%s/", host.KeyClientStorePrefix, clientID))
+	clientPrefix := host.PrefixedClientStoreKey([]byte(clientID))
```

Comment:
seems like these are not quite the same, there's a trailing slash here but not in `host.PrefixedClientStoreKey()`

Reply:
Ahhh, yeah! I will revert then, but I wonder then why that trailing slash is needed there.. 🤔

Reply:
I'm not 100% as I didn't look super deep into it. I tried adding the trailing slash to [PrefixedClientStorePath](https://github.com/cosmos/ibc-go/blob/main/modules/core/24-host/keys.go#L48) but had failing tests elsewhere.